plugins/node/opentelemetry-instrumentation-hapi/README.md

```diff
@@ -66,6 +66,10 @@ The dev dependency of `@hapi/podium@4.1.1` is required to force the compatible t
 
 This package provides automatic tracing for hapi server routes and [request lifecycle](https://github.com/hapijs/hapi/blob/master/API.md#request-lifecycle) extensions defined either directly or via a Hapi plugin.
 
+## Semantic Conventions
+
+This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
+
```

Comment:
I guess it would be nice to have a table of used attrs here?

Reply:
done